src/cli/cli.go

```diff
@@ -0,0 +1,110 @@
+package cli
+
+import (
+	"encoding/base64"
+	"flag"
+	"fmt"
+	"os"
+	"runtime"
+	"strconv"
+	"strings"
+
+	"github.com/f0o/turbo-pancake/common"
+	"github.com/f0o/turbo-pancake/utils"
+	"github.com/f0o/turbo-pancake/worker"
+)
+
+//spawnWorkers : Wrapper that creates Worker-Goroutines
+func spawnWorkers() {
+	for w := 0; w < *common.Threads; w++ {
+		uuid, _ := utils.UUIDv4()

```

Comment:
The only line that calls 
utils.UUIDv4()

but only is using single return in a double return context. maybe second return should be removed or checked for errors

Reply:
error return is implemented for future use - all of this unstable WIP